references/detection/transforms.py

```diff
@@ -437,3 +439,78 @@ def forward(
                 )
 
         return image, target
+
+
+class SimpleCopyPaste(torch.nn.Module):
+    def __init__(self, jittering_type: str = "LSJ"):
```

Comment:
@lezwon feel free to ignore this for now but note that we shouldn't make SimpleCopyPaste use `ScaleJitter` and the other transforms internally. Instead those transforms can be assumed to have been applied before hand. This will allow you to decouple them.

Again feel free to ignore this for as long as you experiment. We can review the design once you are happy with the implementation. I just thought to give you an early heads up.

Reply:
@datumbox sure. will keep that in mind :) Thanks